charts/osm/crds/multiclusterservice.yaml

```diff
@@ -58,6 +58,12 @@ spec:
                       name:
                         description: Name of the remote cluster
                         type: string
+                      weight:
+                        description: Load balancing weight of the remote cluster
+                        type: integer
+                        minimum: 1
+                        maximum: 10000
```

Comment:
Any particular reason why this is capped at 10000 ?

Reply:
No, I confused myself with another number.

Anyway, in Envoy, there is no limit on the load balancing weight. If we follow convention, we can do the same. https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/endpoint/v3/endpoint_components.proto#envoy-v3-api-field-config-endpoint-v3-localitylbendpoints-load-balancing-weight